src/ontology/mondo-edit.obo

```diff
@@ -400975,19 +400959,37 @@ intersection_of: disease_has_location UBERON:0009853 ! body of uterus
 id: MONDO:0021651
 name: synpolydactyly
 def: "A joint presentation of syndactyly (fusion of digits) and polydactyly (production of supernumerary digits)." [https://en.wikipedia.org/wiki/Synpolydactyly]
+subset: gard_rare
+subset: ordo_morphological_anomaly {source="Orphanet:93403"}
 synonym: "polysyndactyly" EXACT [NCIT:C125597, SCTID:84598000, UMLS:C0265553]
+synonym: "SPD" RELATED ABBREVIATION [GARD:0005087]
+synonym: "syndactyly type 2" EXACT [MONDO:0019683]
+synonym: "synpolydactyly" EXACT [Orphanet:93403]
 synonym: "synpolydactyly" EXACT [NCIT:C75003]
+xref: GARD:0005087 {source="MONDO:equivalentTo"}
+xref: ICD10:Q70.0 {source="ORDO:93403/nd", source="ORDO:93403/attributed", source="Orphanet:93403"}
+xref: ICD10:Q70.2 {source="ORDO:93403/nd", source="ORDO:93403/attributed", source="Orphanet:93403"}
+xref: MESH:C538153 {source="MONDO:subClassOf", source="ORDO:93403/e", source="Orphanet:93403"}
 xref: MESH:C538153 {source="MONDO:equivalentTo"}
 xref: NCIT:C125597 {source="MONDO:relatedTo"}
 xref: NCIT:C75003 {source="MONDO:equivalentTo"}
+xref: OMIM:186000 {source="ORDO:93403/btnt", source="MONDO:superClassOf", source="Orphanet:93403", source="GARD:0005087"}
+xref: OMIM:608180 {source="ORDO:93403/btnt", source="MONDO:superClassOf", source="Orphanet:93403"}
+xref: OMIM:610234 {source="ORDO:93403/btnt", source="MONDO:superClassOf", source="Orphanet:93403"}
+xref: Orphanet:93403 {source="MONDO:equivalentTo", source="GARD:0005087"}
+xref: SCTID:715724002 {source="MONDO:equivalentTo"}
 xref: SCTID:84598000 {source="MONDO:equivalentTo"}
 xref: UMLS:C0265553 {source="MONDO:equivalentTo"}
+xref: UMLS:C2699746 {source="MONDO:notFoundInDiseaseSubset", source="ORDO:93403/e", source="Orphanet:93403"}
 xref: UMLS:C2699746 {source="MONDO:notFoundInDiseaseSubset", source="NCIT:C75003"}
 is_a: MONDO:0018454 {source="MONDO:0018455-obsoleted"} ! dysostosis of genetic origin
 intersection_of: MONDO:0018454 ! dysostosis of genetic origin
 intersection_of: disease_has_major_feature HP:0001159 ! Syndactyly
 intersection_of: disease_has_major_feature HP:0010442 ! Polydactyly
 relationship: excluded_subClassOf MONDO:0002254 {source="NCIT:C75003"} ! syndromic disease
+relationship: excluded_subClassOf MONDO:0019530 {source="Orphanet:93403"} ! non-syndromic syndactyly
```

Comment:
@sabrinatoro note, I excluded this superclass, it is no longer a superclass in Orphanet.

Reply:
ok